config/application.rb

```diff
@@ -18,6 +18,7 @@ class Application < Rails::Application
 
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**", "*.{rb,yml}")]
+    config.autoload_paths += Dir[Rails.root.join("app", "repositories", "*.rb")]
```

Comment:
@timurvafin do you think it's a good place to load files if they are not loaded by some gem?